src/UserAgreement/Quiz.less

```diff
@@ -0,0 +1,88 @@
+@import '~@gen3/ui-component/dist/css/base.less';
+
+.quiz__form {
+  display: block;
+  width: 1025px;
```

Comment:
The quiz looks off center in my browser, I think we could make it's width responsive to fix this (ie. `100%`)

Reply:
will check with Vivi, because the design comes from her :)